spec/index.html

```diff
@@ -533,21 +533,114 @@ <h2>Basic Concepts</h2>
       interoperability considerations.
     </p>
 
-    <p>
-      This is because YAML is more flexible than [[JSON]]:
-    </p>
+    <section id="json-vs-yaml">
+      <h3>JSON vs YAML comparison</h3>
 
-    <ul>
-      <li>YAML supports different encodings, including UTF-8, UTF-16, and UTF-32.</li>
-      <li>YAML supports more native data types than JSON.</li>
-      <li>The structure of a <a>YAML document</a> &mdash; that is, a <a>YAML representation graph</a> &mdash;
-      is a rooted, directed graph that can have cycles.</li>
-      <li>YAML is serialized as a <a>YAML stream</a>, which is a sequence of <a>YAML documents</a>.
-          While a stream often contains one document,
-          streams with multiple documents are used to aggregate multiple,
-          related, documents into a single file or network stream.
-      </li>
-    </ul>
+      <p>YAML is more flexible than JSON, as illustrated by comparison table below.</p>
+
+      <table class="simple">
+        <thead>
+        <tr>
+          <th>Features</th>
+          <th>[[JSON]]</th>
+          <th>[[YAML]]</th>
+        </tr>
+        </thead>
+        <tbody>
+        <tr>
+          <th colspan="3">Allowed encodings</th>
+        </tr>
+
+        <tr>
+          <th>UTF-8</th>
+          <td><a data-cite="JSON#section-8.1">✔️</a></td>
+          <td><a data-cite="YAML#52-character-encodings">✔️</a></td>
+        </tr>
+
+        <tr>
+          <th>UTF-16</th>
+          <td><a data-cite="JSON#section-8.1">❌</a></td>
+          <td><a data-cite="YAML#52-character-encodings">✔️</a></td>
+        </tr>
+
+        <tr>
+          <th>UTF-32</th>
+          <td><a data-cite="JSON#section-8.1">❌</a></td>
+          <td><a data-cite="YAML#52-character-encodings">✔️</a></td>
+        </tr>
+
+        <tr>
+          <th colspan="3">Native data types</th>
+        </tr>
+
+        <tr>
+          <th><code>{}</code> object</th>
+          <td><a data-cite="JSON#section-4">✔️</a></td>
+          <td><a data-cite="YAML#3211-nodes">✔️</a></td>
+        </tr>
+
+        <tr>
+          <th><code>[]</code> array</th>
+          <td><a data-cite="JSON#section-5">✔️</a></td>
+          <td><a data-cite="YAML#3211-nodes">✔️</a></td>
+        </tr>
+
+        <tr>
+          <th>string</th>
+          <td><a data-cite="JSON#section-7">✔️</a></td>
+          <td><a data-cite="YAML#3211-nodes">✔️</a></td>
+        </tr>
+
+        <tr>
+          <th>number</th>
+          <td><a data-cite="JSON#section-6">✔️</a></td>
+          <td>✔️<a data-cite="YAML#10213-integer">integer</a> <a data-cite="YAML#10214-floating-point">floating
```

Comment:
Maybe some punctuation, too... depends partly on whether "floating point" is always on one line. (Don't want people to read "integer floating point" as one thing.)
```suggestion
          <td>✔️<a data-cite="YAML#10213-integer">integer</a> <br> <a data-cite="YAML#10214-floating-point">floating
```